tests/compile-fail/env_args.rs

```diff
@@ -0,0 +1,6 @@
+//error-pattern: no mir for `std
```

Comment:
My `mir-for-all` branch of rustc gets to here:

```
<core macros>:2:1: 1:1 error: can't call C ABI function: pthread_mutex_lock
src/libstd/sys/common/mutex.rs:41:33: 41:46 note: inside call to std::sys::mutex::Mutex::lock
```

:smiley: 

I need to prepare that PR for enabling this in rustc optionally soon.